doc/nrf/app_dev/device_guides/nrf54l/cryptography.rst

```diff
@@ -0,0 +1,307 @@
+| ``key_type`` (``psa_set_key_type``)          | A supported key type.                                                       | Sets the key type and size.                                                                                                               |
+|                                              |                                                                             |                                                                                                                                           |
+|                                              |                                                                             | See :ref:`ug_nrf54l_crypto_kmu_supported_key_types` for overview of the supported key types for each driver.                              |
++----------------------------------------------+-----------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+| ``key_bits`` (``psa_set_key_bits``)          | A supported key size for the key type.                                      | Sets the key type and size.                                                                                                               |
+|                                              |                                                                             |                                                                                                                                           |
+|                                              |                                                                             | See :ref:`ug_nrf54l_crypto_kmu_supported_key_types` for overview of the supported key types for each driver.                              |
++----------------------------------------------+-----------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------+
+| ``key lifetime`` (``psa_set_key_lifetime``)  | ``PSA_KEY_LIFETIME_FROM_PERSISTENCE_AND_LOCATION(persistence, location)``   | ``CRACEN_KEY_PERSISTENCE_REVOKABLE`` is a custom persistence mode, which will revoke the key slots when the key is destroyed.             |
+|                                              | where persistence is set to one of the following:                           |                                                                                                                                           |
+|                                              | ``PSA_KEY_PERSISTENCE_DEFAULT``,                                            |                                                                                                                                           |
```

Comment:
What does default do?

Reply:
From [here](https://github.com/hellesvik-nordic/sdk-nrf/blob/e6eec800a3a7cd53a113ab557bc73ff8523540cd/subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c#L663-L664) we can see that it is LIB_KMU_REV_POLICY_ROTATING. 
The behaviour of this is explained under the OPS -> KMU -> 7.7.3.2 Operations.

Reply:
Added a bit more info on default and read-only persistence.